src/pages/PostDetailWriterPage.jsx

```diff
@@ -28,22 +37,68 @@ const PostDetailWriterPage = () => {
     }).then((result) => {
       if (result.isConfirmed) {
         // 공고 수정하게 됨
+        navigate('/post-write');
       } else if (result.isDenied) {
         postDelete();
       }
     });
   };
+
+  // 피커정보
+  // eslint-disable-next-line
+  const pickerInfo = (isMatch) => {
+    if (isMatch) {
+      return (
+        <div>
+          <div className="my-8">
+            <div className="text-2xl text-blue py-2">피커정보</div>
+            <div className="flex">
+              <div className="text-zinc-400">도착시간</div>
+              <div className="ml-5">3000원</div>
+            </div>
+            <div className="flex my-1">
+              <div className="text-zinc-400">계좌정보</div>
+              <div className="ml-5">농협 01010101010101010101</div>
+            </div>
+            <div className="flex my-1">
+              <div className="text-zinc-400">계좌정보</div>
+              <div className="ml-5">010-1234-5678</div>
+            </div>
+          </div>
+        </div>
+      );
+    }
+  };
+
+  // eslint-disable-next-line
+  const threeDot = (isMatch) => {
+    if (!isMatch) {
+      return <BsThreeDotsVertical size="25" style={{ color: 'white' }} />;
+    }
+  };
+
+  const matching = () => {
+    if (isMatch) {
+      console.log(isMatch);
+      return (
+        <div>
+          <div className="text-white text-lg">피커가 픽업을 시작했어요!</div>
+          <div className="text-white text-lg">피커에게 픽업팁을 송금 해주세요.</div>
+        </div>
```

Comment:
```suggestion
        <div className="text-white text-lg">
          <div>피커가 픽업을 시작했어요!</div>
          <div>피커에게 픽업팁을 송금 해주세요.</div>
        </div>
```

동일한 스타일을 반복하지 않고 이렇게 쓸 수도 있을 것 같아요!